backend/src/app.rs

```diff
@@ -1082,6 +1083,34 @@ mod test {
         assert!(res.is_err());
     }
 
+    #[tokio::test]
+    #[tracing_test::traced_test]
+    async fn test_event_create_email_fail_validation() {
+        let app = App::new(
+            Arc::new(InMemoryEventsDB::default()),
+            Arc::new(PubSubInMemory::default()),
+            Arc::new(MockViewers::new()),
+            Arc::new(Payment::default()),
+            Tracking::default(),
+            String::new(),
+        );
+
+        let res = app
+            .create_event(AddEvent {
+                data: EventData {
+                    name: String::from("123456789"),
+                    description: String::from("123456789 123456789 123456789 !"),
```

Comment:
i think we have `TEST...` constants for this (in case we ever change the validation we only have to update these

Reply:
I can only see one for a TEST_VALID_QUESTION, I copied this template from the function below it

Reply:
Updated to use them now 👍